.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,120 @@
+    options:
+    - label: I have searched the existing issues
+      required: true
+
+- type: checkboxes
+  attributes:
+    label: Have you tried deactivating all plugins except Gutenberg?
+    description: Please make sure you have ruled out plugin conflicts before reporting.
+    options:
+    - label: I have tested with all plugins deactivated. 
+      required: true
+      
+- type: checkboxes
+  attributes:
+    label: Have you tried replicating the bug using a default theme e.g. Twenty Twenty?
+    description: Please make sure you have confirmed it's not a theme specific problem.
+    options:
+    - label: I have tested with a default theme.
+      required: true
+
+- type: textarea
+  attributes:
+    label: Description
+    description: Please write a brief description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Step-by-step reproduction instructions
+    description: Please list the steps needed to reproduce the bug. 
+    placeholder: |
+      1. Go to '...'
+      2. Click on '...'
+      3. Scroll down to '...'
+      4. See error...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: Please describe what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Screenshots or screen recording (optional)
+    description: |
+     If possible, please upload a screenshot or screen recording which demonstrates the bug. You can use LIEcap to create a GIF screen recording: https://www.cockos.com/licecap/
+      
+    Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    required: false
+
+- type: textarea
+  attributes:
+    label: Code snippet (optional)
+    description: |
+     If this bug is to related to a developer API, please share a code snippet that demonstrates the issue. For small snippets paste it directly here, or you can use GitHub Gist to share multiple code files: https://gist.github.com
+     Please ensure the shared code can be used by a developer to reproduce the issue—ideally it can be copied into a local development environment or executed in a browser console to help debug the issue.
+    required: false
+
+- type: textarea
+  attributes:
+    label: WordPress Information
+    description: |
+     Please list what WordPress version you are using. You can find this information in Tools → Site Health → Info → WordPress
+    required: false
+    
+- type: textarea
+  attributes:
+    label: Gutenberg Information
+    description: |
+     Please list what Gutenberg version you are using. If you aren't using Gutenberg, please note that it's not installed.
+    required: false
+    
+  - type: dropdown
```

Comment:
I played with the dropdown idea as well and found it is more restrictive than convenient, as reports can't include:
- The browser version (e.g., is it Legacy Edge or chrome-based Edge?)
- The actual browser if it's not listed and the user selects "Other".

In the same line as my other comment, I think a single "Device Information" text area would yield better results.